libs/transactions/src/lib/components/transaction-item/transaction-item.component.ts

```diff
@@ -20,7 +18,7 @@ export class TransactionItemComponent implements OnChanges {
   @Input() transaction!: TransactionItem;
   public amount = 0;
   public isAmountPositive = true;
-  public additionsDetailsTpl: TemplateRef<AdditionalDetailsContext> | undefined;
```

Comment:
I suggest not to use `public additionsDetailsTpl: TemplateRef<AdditionalDetailsContext> | undefined;` here, since

1. it would create yet another place where we have to import this context's interface
2.this type is correctly inferred from the (typed) config service